tools/metadata/validate.py

```diff
@@ -99,9 +99,11 @@ def main():
   if args.files:
     _ = [validate_file(file) for file in args.files if filter_files(file)]
   else:  # Run all
-    for root, _, files in os.walk(os.getcwd()):
-      for file in [f for f in files if filter_files(os.path.join(root, f))]:
-        validate_file(os.path.join(root, file))
+    args = ['git', 'ls-files']
```

Comment:
Please verify this works on Windows checkouts from powershell and cmd shell before submitting. 

And/or move to a separate function so that it can be run through a unit test, so it gets automatically checked on Windows too.